### PR TITLE
git-reauthor: updated documentation to show rename workaround

### LINKED ISSUE #752: [git reauthor] update documentation to show how to replace name
This issue will show how to change the name using the current commands, until a permanent fix is put in place using a name parameter.

### PR DESCRIPTION
**Pull request by tiemonl**
closes #752.
@spacewander please review.

### CODE REVIEW

File: man/git-reauthor.md [hunk 1]
```diff
@@ -66,6 +66,10 @@ Change only the committer email of Jack (keep the author email already used)
 
     $ git reauthor --old-email jack@perso.me --correct-email jack@work.com --type committer
 
+Change only the name and keep email to merge one user with multiple pseudonyms.
+
+    $ git reauthor --old-email jack@perso.me --correct-email jack@perso.me --correct-name Jack
```
Comment (spacewander) [inline man/git-reauthor.md]: The `--correct-email jack@perso.me` part is useless. If you want to keep the email part, use `--correct-name blahblah` is enough.

Comment (tiemonl) [reply man/git-reauthor.md]: Good catch. Makes it simpler too. Change made.

**Comment by spacewander** [comment]
@tiemonl 
Could you squash your commits so that I merge it?

**Comment by tiemonl** [comment]
@spacewander I haven't squashed before, but it looks like I did it right after a quick trial and error. I believe it should be good to go now.

**Comment by spacewander** [comment]
Merged. Thank you!

**Pull request merged**